### DISCUSSION TITLE
Welcome to Discussions

### DISCUSSION THREAD

**Discussion by dillof** [nih-at/libzip#460] [Announcements]
If you have questions about libzip or its usage, feel free to start a discussion here, which makes them easier to find by others in the future. 

We would prefer to keep issues for actual bug reports and change requests.

**Comment by mihaly-sisak** [comment]
Hi all! Currently profiling my game, loading save files (reading zip files). 
Perf flame graph shows me that ~34% of runtime is inflate_fast(), and ~28% of runtime is crc32_little(). 
Is there any option to skip crc check while reading (zlib) zip files? 
Tried to google the issue and looked at docs, could not find anything.

**Reply by Giandj** [reply]
Hi, i'm not sure on Libzip3 but on my Linux Mint 19 I have Libzip4 installed, but Mint continue to use libzip 1.1.21...
Is there a method to install Libzip3 or Libzip4 and check that everything is allright ?
Normally I have installed "sudo apt-get install libzip4", but Linux Mint continue to use 1.1.21,
for example when i try to compile "hypseus-singe" it use oldest version of libzip, not latest, during compiling "make" process....
Why ?

**Reply by 0-wiz-0** [reply]
@mihaly-sisak No, there is no option to skip the CRC check. I've read that [zlib-ng](https://github.com/zlib-ng/zlib-ng) is faster than zlib, so you could try compiling libzip against that and see if it helps.

**Comment by 0-wiz-0** [comment]
@Giandj We're not using Linux Mint, and I couldn't find information about the libzip packages on Linux Mint.
I see that Mint is based on Ubuntu, which is based on Debian, so you could try installing the [Debian package](https://sources.debian.org/src/libzip/1.11.3-1/) for the latest version. If you have problems with that, a forum dedicated to Linux Mint is the best place to ask questions.
